package.json

```
@@ -96,20 +96,22 @@
     "typescript": "^3.7.2"
   },
   "dependencies": {
-    "@serialport/bindings": "3",
+    "@serialport/bindings": "^8.0.4",
     "@types/serialport": "^7.0.4",
     "electron-log": "^3.0.9",
     "electron-prompt": "^1.4.0",
     "electron-updater": "^4.1.2",
     "follow-redirects": "^1.7.0",
     "global": "^4.4.0",
-    "hid-listen": "github:zvecr/hid-listen",
     "install": "^0.13.0",
+    "javascript-state-machine": "^3.1.0",
     "lodash": "^4.17.15",
     "node-abi": "^2.12.0",
+    "node-hid": "^0.7.9",
     "path": "^0.12.7",
     "serialport": "^8.0.5",
     "temp": "^0.9.0",
-    "usb": "^1.6.0"
+    "usb": "^1.6.0",
+    "usb-detection": "^4.5.0"
```

```suggestion
    "usb-detection": "^4.7.0"
```

I notice this was changed in the yarn lock but not in package.json